app/models/site_config.rb

```diff
@@ -39,6 +41,11 @@ class SiteConfig < RailsSettings::Base
     members: ApplicationConfig["DEFAULT_EMAIL"]
   }
 
+  field :navigation, type: :array, default: [
+    { name: 'Reading List', url: 'abc.com', icon: LISTING_ICON },
+    { name: 'Listing', url: 'def.com', icon: MIC_ICON }
```

Comment:
can the URLs be empty?

Reply:
This is a good question, technically they shouldn't be empty but it would not break the interface if they are. We will have a set of default urls and navigation elements as a start.  

What are your thoughts?

Reply:
maybe we can default them to the actual `/readinglist` and `/listings` ? 

ps. `Listing` should be plural: `Listings`

Reply:
Haha yes definitely, this PR was just a spike to show that in theory hooking all this up works. I will definitely use sensible names, values and urls. The defaults will be as per the current dev website, not my random values  😅